src/main/java/com/projecturanus/betterp2p/client/render/OutlineRenderer.java

```diff
@@ -15,14 +15,10 @@
 import kotlin.Pair;
 
 /**
- * Everything in this inner class is From a mod that has MIT license owned by romelo333 and maintained by McJty
- *
- * License is here: https://github.com/romelo333/notenoughwands1.8.8/blob/master/LICENSE
- *
- * Specific source of code from the GenericWand class:
- * https://github.com/romelo333/notenoughwands1.8.8/blob/20952f50e7c1ab3fd676ed3da302666295e3cac8/src/main/java/romelo333/notenoughwands/Items/GenericWand.java
- *
- * Version 1.1: Edited to support rotation
+ * This class is based on the <a href=
+ * "https://github.com/romelo333/notenoughwands1.8.8/blob/20952f50e7c1ab3fd676ed3da302666295e3cac8/src/main/java/romelo333/notenoughwands/Items/GenericWand.java">GenericWand</a>
+ * class from a mod with MIT license.
+ * <a href="https://github.com/romelo333/notenoughwands1.8.8/blob/master/LICENSE">Source license</a>
```

Comment:
Not a big fan of using javadoc tags, use KDocs instead

Reply:
> https://kotlinlang.org/docs/kotlin-doc.html#see-identifier

Just in case because the `@see` tag was linked specifically, I'll note here why I didn't use it.  
According to the link you provided, the See tag is for linking the methods in the program ([example](https://www.kodeco.com/30067669-documenting-kotlin-code-for-android-using-kdoc-and-dokka#toc-anchor-007)). The documentation in my commit refers to the external methods, so it is an external link that can be written as `[text](url)`.

I will apply the fix shortly.

Reply:
Seems like `[test](https://example.com)` doesn't render in IntelliJ. It tried to render the following code
```
 * This class is based on the
 * [test](https://example.com)
 * TEST
```
The result was as follows
![image](https://github.com/GTNewHorizons/BetterP2P/assets/73699379/df0846a6-2061-4926-8c12-056a87c771f0)

Reply:
Seems like IntelliJ doesn't support markdown in docs. Here's the [ticket](https://youtrack.jetbrains.com/issue/IDEA-109829/Support-Markdown-in-Javadoc) about it.